project_generator/project.py

```diff
@@ -361,48 +364,33 @@ def flash(self, tool):
             project_files = [os.path.join(proj_dic['output_dir']['path'], proj_dic['name'])]
         flash(flasher, proj_dic, self.name, project_files, tool, self.workspace.settings)
 
-    def export(self, tool, copy):
+    def generate_dic(self, tool, copy):
```

Comment:
Why the name change? It is a little confusing when generate_dict_for_tool already exists